src/Field_One.php

```diff
@@ -132,9 +132,21 @@ public function ref($defaults = [])
         $m = $this->getModel($defaults);
         if ($this->owner->loaded()) {
             if ($this->their_field) {
-                return $m->loadBy($this->their_field, $this->owner[$this->our_field]);
+                return $m->tryLoadBy($this->their_field, $this->owner[$this->our_field])
+                    ->addHook('afterSave', function ($m) {
```

Comment:
OK, i think i want to keep as-is for the following pattern:

$user->ref('contact_id')->load(123)->save();

which allows to easily update to a different record.